codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/testutil/TestHelpers.kt

```diff
@@ -94,6 +94,14 @@ fun String.asSmithyModel(sourceLocation: String? = null, smithyVersion: String =
         .unwrap()
 }
 
+fun createModelFromLines(vararg lines: String): Model {
+    var source = lines.joinToString(separator = "\n")
+    return Model.assembler()
+        .addUnparsedModel("test.smithy", source)
+        .assemble()
+        .unwrap()
+}
+
```

Comment:
We already have a `String.asSmithyModel()` extension function: https://github.com/awslabs/smithy-rs/blob/d81bf3c47a4ce35a2b6c66611323057f7e30181e/codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/testutil/TestHelpers.kt#L91